apps/src/code-studio/components/aiTutor/aiTutorFloatingActionButton.tsx

```diff
@@ -1,28 +1,32 @@
 import React, {useState} from 'react';
-import {Provider} from 'react-redux';
+import {Provider, useSelector} from 'react-redux';
 import {getStore} from '@cdo/apps/redux';
 import style from './ai-tutor.module.scss';
 import aiFabIcon from '@cdo/static/ai-fab-background.png';
 import AITutorPanel from './aiTutorPanel';
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
-import {levelShape} from './aiTutorShapes';
+import {AITutorState} from '@cdo/apps/aiTutor/redux/aiTutorRedux';
 
 /**
  * Renders an AI Bot icon button in the bottom left corner over other UI elements that controls
  * toggling the AI Tutor Panel open and closed.
  */
-const AITutorFloatingActionButton = ({level}) => {
+
+const AITutorFloatingActionButton: React.FunctionComponent = () => {
   const store = getStore();
   const [isOpen, setIsOpen] = useState(false);
+  const level = useSelector(
```

Comment:
(can be a follow up) we now have [useAppSelector](https://github.com/code-dot-org/code-dot-org/blob/7788d30a9e067ee634e17002998ca4da9d866557/apps/src/util/reduxHooks.ts#L14) which makes this syntax much nicer!